tests/ui-nightly/invalid-impls/invalid-impls.stderr

```diff
@@ -25,13 +25,13 @@ help: consider restricting type parameter `T`
 26 | impl_or_verify!(T: zerocopy::TryFromBytes => TryFromBytes for Foo<T>);
    |                  ++++++++++++++++++++++++
 
-error[E0277]: the trait bound `T: zerocopy::FromZeros` is not satisfied
+error[E0277]: the trait bound `T: FromZeroes` is not satisfied
```

Comment:
Blegh, *seriously*? @joshlf This sort of error is why we didn't do the renaming in 0.7. Given that it's unavoidable, perhaps we cut a final release of 0.7 with the rename + deprecated re-exports, and then in 0.8 we don't provide the re-exports so our error messages are sensible.

Reply:
Ah I know what the issue is. We're not passing the new __INTERNAL_ONLY cfg in our rollers. We should maybe use cargo.sh in the rollers too.

In theory that sounds like a good plan, but wouldn't that be a breaking change for those with warnings denied?

Reply:
Yes, it would break builds with warnings denied, but a [broken workflow is not a breaking change](https://xkcd.com/1172/). The Cargo reference explicitly [defines deprecations as compatible changes](https://doc.rust-lang.org/cargo/reference/semver.html#new-lints):
> Some changes to a library may cause new lints to be triggered in users of that library. This should generally be considered a compatible change. [...] Denying warnings should be done with care and the understanding that new lints may be introduced over time. However, library authors should be cautious about introducing new warnings and may want to consider the potential impact on their users.

We've exercised caution here and tried valiantly to avoid deprecating the items in 0.7. But, if we don't do so in 0.7, our users in 0.8 are going to be cursed to endure actively misleading error messages.

Reply:
Alternatively, we can just do the rename without providing re-exports. Maybe the names are close enough that Rust's suggestions heuristics will suggest the new names when the old ones are used.